app/workers/concerns/jittered_retry_in.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module JitteredRetryIn
+  extend ActiveSupport::Concern
+
+  included do
+    sidekiq_retry_in do |count|
+      # This is the same as the default Sidekiq implementation of delay_for,
+      # but with a random 50% more jitter.
+      (15 + (count)**4) * (1 + rand(0..0.5)).floor
```

Comment:
Not a ruby expert but i think the `.floor` here applies to `(1 + rand(0..0.5)` (i quickly checked in `irb` to be sure) which will thus always return `1` so this is a multiplication by 1. if you wanted to ensure the whole expression returns an int, it's better to apply it after the multiplication.
```suggestion
      ((15 + (count)**4) * (1 + rand(0..0.5))).floor
```

Reply:
Yes, thank you! I removed some parentheses and forgot to double-check 🤦‍♂️  Fixed now!